src/metabase/api/search.clj

```diff
@@ -558,12 +559,30 @@
                              :offset              mw.offset-paging/*offset*
                              :search-native-query search_native_query
                              :verified            verified}))
-        duration   (- (System/currentTimeMillis) start-time)]
-    ;; Only track global searches
-    (when (and (nil? models)
-               (nil? table_db_id)
-               (not archived))
-      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id* {:runtime-milliseconds duration}))
+        duration   (- (System/currentTimeMillis) start-time)
+        has-advanced-filters (some some?
+                                   [models created_by created_at last_edited_by
+                                    last_edited_at search_native_query verified])]
+    (when (not (nil? context))
+      (cond
+        (and
+         (nil? table_db_id)
+         (not archived))
+        (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id*
+                               {:runtime-milliseconds duration
+                                :context              context}))
+      (cond
+        has-advanced-filters
+        (snowplow/track-event! ::snowplow/search-results-filtered api/*current-user-id*
+                               {:runtime-milliseconds  duration
+                                :content-type          (u/one-or-many models)
```

Comment:
why isn't this just called models? like
```suggestion
                                :models          (u/one-or-many models)
```

Reply:
The keys match the labels of the search filters - in hindsight the keys should probably have been uniform throughout but that's a conversation for another time